pkg/rancher-desktop/components/Help.vue

```diff
@@ -0,0 +1,110 @@
+<script lang="ts">
+import { shell } from 'electron';
+import Vue from 'vue';
+import { mapState } from 'vuex';
+
+export default Vue.extend({
+  name:       'help',
+  props:      {
+    fixedUrl: {
+      type:     String,
+      default:  null,
+    },
+    tooltip: {
+      type:    String,
+      default: null,
+    },
+  },
+  computed: {
+    ...mapState('help', ['url']),
+    helpUrl(): string {
+      return this.fixedUrl ?? this.url;
+    },
+    tooltipContent(): string | null {
+      return this.helpUrl ? this.tooltip : null;
+    },
+  },
+  methods:  {
+    openUrl() {
+      if (this.helpUrl) {
+        shell.openExternal(this.helpUrl);
+      }
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="help-button">
+    <button
+      v-tooltip="{
+        content: tooltipContent,
+        placement: 'right'
+      }"
+      class="btn role-fab ripple"
+      :class="{
+        disabled: !helpUrl
+      }"
+      @click="openUrl"
+    >
+      <span
+        class="icon icon-question-mark"
+        :class="{
+          disabled: !helpUrl
+        }"
+      />
```

Comment:
NOTE: The design of this looks very similar to `pkg/rancher-desktop/components/Preferences/ButtonOpen.vue`. We will want to consider if it makes sense to extract out to a generic component that can be used by others if this is a pattern that we find ourselves using more often in the future.